README.md

```diff
@@ -135,6 +135,22 @@ const didWithCap = did.withCapability(cacao)
 const didWithCap2 = new DID({provider, resolver: KeyResolver.getResolver(), capability: cacao})
 ```
 
+## Security Considerations
+
+Ceramic allows for keys attached to DIDs to continue making updates to streams after revocation for a certain grace period. This is done to ensure that if multiple updates were committed to different nodes before the key was revoked, but the nodes chose to anchor commits at different times, the updates still make it through.
```

Comment:
huh, this just made me realize.  Did we implement the key revocation grace period in a way that applies to all DIDs, not just 3IDs?  That seems like a mistake, the issues with key revocations and anchor timings are pretty specific to 3IDs.  @oed

Reply:
I would say it applies to all DIDs since things signed right before the revocation might not be anchored yet. This is also relevant for CACAO, which I just realized doesn't use the revocation timeout yet:  https://github.com/ceramicnetwork/js-did/blob/main/src/did.ts#L334